edweiss-app/app/(app)/lectures/slides/index.tsx

```diff
@@ -25,13 +25,12 @@ import LectureDisplay from '@/model/lectures/lectureDoc';
 import { useLocalSearchParams } from 'expo-router';
 import * as ScreenOrientation from 'expo-screen-orientation';
 import React, { useEffect, useState } from 'react';
-import { ActivityIndicator, Dimensions, Linking } from 'react-native';
+import { ActivityIndicator, Dimensions, DimensionValue } from 'react-native';
 import Pdf from 'react-native-pdf';
 
 // Types
 type Lecture = LectureDisplay.Lecture;
 
-
 // ------------------------------------------------------------
 // -------------------  Remote Control Screen  ----------------
```

Comment:
A minor detail, I would prefer like to see in the comment section `Lecture Screen` 😉. The remote control is in `lectures/remotecontrol/index.tsx` 😁. Just as a matter to be consistent.

Reply:
Oh yes well noticed, I must have taken it by mistake when I was fixing some issues with the parameters thank you

Reply:
No worries, I am happy to hear that you found this useful 🚀.